### ISSUE TITLE
Blueprint methods missing explicit category specifier

### ISSUE THREAD

**Issue by mwojt-platige** [Mystfit/Unreal-StableDiffusionTools#5]
I'm getting a lot of similar errors when compiling the plugin:
```
 Error: An explicit Category specifier is required for Blueprint accessible functions in an Engine module.
```
My workaround was to add the `Category = "StableDiffusion"` to all `UPROPERTY` and `UFUNCTION` decorators in the headers.

**Comment by Mystfit** [comment]
Thanks, I'll look into it. Oddly enough that doesn't seem to crash on my end - not sure why.

**Comment by Mystfit** [comment]
Just realized why I haven't received that error. I've been running my plugin in my project's plugins folder, rather than from the engine's plugins folder. My bad!

This is good to know for when I eventually submit this to the marketplace.

**Comment by mwojt-platige** [comment]
Oh yes, I totally forgot to mention that I'm building this plugin into the engine!

**Comment by Mystfit** [comment]
Fixed in v0.3.1
